Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -415,6 +415,17 @@ class CallMethod: Operation {
     }
 }
 
+class CallComputedMethod: Operation {
+    var numArguments: Int {
+        return numInputs - 2
+    }
+
+    init(numArguments: Int) {
+        // reference object is the first input and method name is the second input
+        super.init(numInputs: numArguments + 2, numOutputs: 1, attributes: [.isParametric, .isVarargs, .isCall])
```

Comment:
I think this is now no longer `.isParametric`

Reply:
The rough rule of thumbs is that every Operation class that has members other than those already in the `Operation` class is parametric. For example integer values (`LoadInteger`), string values (`LoadProperty` and `CallMethod`), or Arrays (`CallFunctionWithSpread`). Maybe a better definition is that an operation should be parametric if it can/should be mutated by the [OperationMutator](https://github.com/googleprojectzero/fuzzilli/blob/master/Sources/Fuzzilli/Mutators/OperationMutator.swift). The `CallComputedMethod` class now no longer has any members and there is no obvious way in which the OperationMutator could meaningfully mutate it.

Reply:
great, thanks for that explanation that's very helpful. Shall I add this explanation to the comments under Instruction.swift just so that we have this knowledge is captured in the source?

Reply:
Yep nice, thanks!

Reply:
Now you just need to remove the `.isParametric` here and I'll merge it ;D

Reply:
woops! all done now :)